app/src/main/java/com/tari/android/wallet/ui/activity/CreateWalletActivity.kt

```diff
@@ -0,0 +1,208 @@
+package com.tari.android.wallet.ui.activity
+
+import android.animation.*
+import android.os.Bundle
+import android.os.Handler
+import android.view.View
+import android.view.ViewTreeObserver
+import android.widget.*
+import androidx.core.view.marginTop
+import butterknife.BindDimen
+import butterknife.BindView
+import butterknife.OnClick
+import com.airbnb.lottie.LottieAnimationView
+import com.daasuu.ei.Ease
+import com.daasuu.ei.EasingInterpolator
+import com.tari.android.wallet.R
+import com.tari.android.wallet.ui.util.UiUtil.getResourceUri
+import com.tari.android.wallet.util.Constants
+
+import java.lang.ref.WeakReference
+
+class CreateWalletActivity : BaseActivity() {
+
+    @BindView(R.id.main_anim_tari)
+    lateinit var tariAnimationView: LottieAnimationView
+    @BindView(R.id.rootView)
+    lateinit var rootView: FrameLayout
+    @BindView(R.id.crypto_wallet_main_txt1)
+    lateinit var cryptoWalletMainText1: TextView
+    @BindView(R.id.crypto_wallet_main_txt2)
+    lateinit var cryptoWalletMainText2: TextView
+    @BindView(R.id.crypto_wallet_des_txt)
+    lateinit var cryptoWalletDesText: TextView
+    @BindView(R.id.create_wallet_btn)
+    lateinit var createWalletButton: TextView
+    @BindView(R.id.create_wallet_btn_layout)
+    lateinit var walletBtnLayout: FrameLayout
+    @BindView(R.id.viewContainer)
+    lateinit var viewContainer: RelativeLayout
+    @BindView(R.id.txt_testnet)
+    lateinit var testnetTextView: TextView
+    @BindView(R.id.main_img_small_gem)
+    lateinit var smallGemImageView: ImageView
+    @BindView(R.id.progress)
+    lateinit var progressBar: ProgressBar
+    @BindView(R.id.videoView_loop1)
+    lateinit var videoView1: VideoView
+    @BindView(R.id.videoView_loop2)
+    lateinit var videoView2: VideoView
+    @BindView(R.id.video_container)
+    lateinit var videoContainer: FrameLayout
+
+    @BindDimen(R.dimen.splash_screen_title_bottom_margin)
+    @JvmField
+    var mainContentShiftMargin = 0
```

Comment:
Resource name is very clear (bottom margin) but property name is different and not as clear.